install.sh

```diff
@@ -81,17 +81,32 @@ sudo ln -s `pwd`/crew $CREW_PREFIX/bin
 mkdir $CREW_LIB_PATH/lib && cd $CREW_LIB_PATH/lib
 wget -N -c $URL/lib/package.rb
 wget -N -c $URL/lib/package_helpers.rb
-#create the device.json file
+
+#create or update the device.json file
+ruby_version='2.0.0p247'
```

Comment:
I still think it makes sense to not hardcode the version.  The whole point of `device.json` is to store the versions of the _*ACTUAL*_ packages.  Why not be consistent?  What if we removed all the hardcoded links to ruby from `install.sh` and extracted from the ruby.rb package instead?  Would that make sense?